torch/quantization/fx/_equalize.py

```diff
@@ -246,11 +251,32 @@ def get_weight_eq_obs(
         return op_node, weight_eq_obs
 
     elif op_node.op == 'call_function':
-        # TODO
-        return None, None
+        weight_node = maybe_get_weight_eq_obs_node(op_node, modules)
+        if weight_node is not None:
+            weight_eq_obs = modules[str(weight_node.target)]
+            assert(isinstance(weight_eq_obs, _WeightEqualizationObserver))
+            return op_node, weight_eq_obs
 
     return None, None
 
+def maybe_get_weight_eq_obs_node(op_node: Node, modules: Dict[str, nn.Module]) -> Optional[Node]:
+    """ Given the operation node, we want to find its weight equalization
+    observer node
+    """
+    assert(op_node.op == 'call_function')
+    assert(isinstance(op_node.args[1], Node))
+    weight_observer_nodes = collect_producer_nodes(op_node.args[1])
```

Comment:
we are using op_node.args[1] should `1` here be configurable depending on the type of op we have? should we use https://github.com/pytorch/pytorch/blob/master/torch/quantization/fx/utils.py#L17?